packages/react-reconciler/src/ReactFiberHooks.new.js

```diff
@@ -1197,14 +1198,83 @@ function pushEffect(tag, create, destroy, deps) {
   return effect;
 }
 
+function getCallerStackFrame(): string {
+  return new Error().stack
+    .split('\n')
+    .slice(3, 4)
+    .join('\n');
```

Comment:
different browsers have different stacks, e.g. Firefox and Safari do not include the error message in the stack, so the line with `Component@file.js:<line>:<column> is at index 2 in those browsers, not index 3 like in Chrome, Node and IE

=> maybe something like https://github.com/stacktracejs/error-stack-parser/blob/master/dist/error-stack-parser.js ?

Reply:
Thanks for reminding me of that! 😅